partial-update-tests/existing/src/main/java/fixtures/bodystring/StringOperationClient.java

```diff
@@ -0,0 +1,298 @@
+     * @return string value when no string value is sent in response payload.
+     */
+    @Generated
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Response<BinaryData> getNotProvidedWithResponse(RequestOptions requestOptions) {
+        return this.serviceClient.getNotProvidedWithResponse(requestOptions);
+    }
+
+    /**
+     * Get value that is base64 encoded.
+     *
+     * <p><strong>Response Body Schema</strong>
+     *
+     * <pre>{@code
+     * byte[]
+     * }</pre>
+     *
+     * @param requestOptions The options to configure the HTTP request before HTTP client sends it.
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @return value that is base64 encoded.
+     */
+    @Generated
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Response<byte[]> getBase64EncodedWithResponse(RequestOptions requestOptions) {
+        return this.serviceClient.getBase64EncodedWithResponse(requestOptions);
+    }
+
+    /**
+     * Get value that is base64url encoded.
+     *
+     * <p><strong>Response Body Schema</strong>
+     *
+     * <pre>{@code
+     * Base64Url
+     * }</pre>
+     *
+     * @param requestOptions The options to configure the HTTP request before HTTP client sends it.
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @return value that is base64url encoded.
+     */
+    @Generated
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Response<BinaryData> getBase64UrlEncodedWithResponse(RequestOptions requestOptions) {
+        return this.serviceClient.getBase64UrlEncodedWithResponse(requestOptions);
+    }
+
+    /**
+     * Put value that is base64url encoded.
+     *
+     * <p><strong>Request Body Schema</strong>
+     *
+     * <pre>{@code
+     * Base64Url
+     * }</pre>
+     *
+     * @param stringBody string body.
+     * @param requestOptions The options to configure the HTTP request before HTTP client sends it.
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @return the response.
+     */
+    @Generated
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Response<Void> putBase64UrlEncodedWithResponse(BinaryData stringBody, RequestOptions requestOptions) {
+        return this.serviceClient.putBase64UrlEncodedWithResponse(stringBody, requestOptions);
+    }
+
+    public String manuallyAddedMethod() {
+        return "manuallyAddedMethod";
+    }
+
+    /**
+     * Get null value that is expected to be base64url encoded.
+     *
+     * <p><strong>Response Body Schema</strong>
+     *
+     * <pre>{@code
+     * Base64Url
+     * }</pre>
+     *
+     * @param requestOptions The options to configure the HTTP request before HTTP client sends it.
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @return null value that is expected to be base64url encoded.
+     */
+    @Generated
```

Comment:
Are we using `@Generated` annotation for `Client` classes only? What about builders? We may manually add some builder options that don't get generated. For e.g. some services support `connectionString` which we don't generate but could be added manually.

Reply:
Currently, we only include `@Generated` for `Client` classes. We can extend to Builders later.

Reply:
I guess the builder, sync and async types should have `@Generated` annotations right from the beginning as those are the most likely user-facing types that will be manually edited.

Reply:
OK, thanks for pointing out, in this issue https://github.com/Azure/autorest.java/issues/1099, we noted down that we will support only for clients as first step, so I did not add to builders. But I will create a issue to add `@Generated` annotation to builders and work on that as my next task. 
Do you think we should block this pr merge until I finish adding the annotation to builders, or we can merge this pr first ? ( I will modify this pr according to the comment and let you review again before merging)

Reply:
issue created: https://github.com/Azure/azure-sdk-for-java/issues/25895

Reply:
At present dev can use `--disable-client-builder` to avoid generate the Builder, if they've done customization. Builder itself is not commonly updated during swagger update.

Handling of Builder will be handled in a next PR. We do not want to have a super big PR for everything.